src/components/rightbar/Rightbar.jsx

```diff
@@ -3,7 +3,6 @@ import React, { useState } from "react";
 import Banner from "../banner/Banner";
 import SearchFilter from "../searchFilter/SearchFilter";
 import Playlist from "../playlist/Playlist";
-import Filters from "../fitlers/Filters";
 
 function Rightbar() {
   const [showFilter, setShowFilter] = useState(false);

```

Comment:
Move this state to SearchFilter component

Reply:
task resolved